autoload/codi.vim

```diff
@@ -140,9 +140,15 @@ if has('unix')
       return ['script', '-qfec', tmp_bin, '/dev/null']
     endfunction
   endif
-else
-  call s:log ('Windows detected, erroring out')
-  call s:err('Codi does not support Windows yet.')
+else  
+  if s:nvim
+    call s:log ('Windows detected')
```

Comment:
Nit: Since this whole branch is "windows detected", can we move it to the outside block?

Also, we should remove the space before `(` (which I think I accidentally left when I first wrote it).

```suggestion
  call s:log('Windows detected')
  if s:nvim
```